app/views/layouts/user_mailer.html.erb

```diff
@@ -46,6 +46,7 @@
                                           attachments['logo.png'].url,
                                           size: '142x19',
                                           style: 'width: 142px; height: 19px;',
+                                          alt: 'Login.gov logo',
```

Comment:
I believe we conventionally use the `APP_NAME` constant for this rather than hardcoding `Login.gov`.
```suggestion
                                          alt: "#{APP_NAME} logo",
```

I noticed that the only place this is hardcoded are the error pages.